woocommerce/woocommerce-gutenberg-products-block
# rename ProductQueryFilters file to match class name (PSR4 compatible)

Fixes #3007

During https://github.com/Automattic/jetpack/pull/16819, one of our classes was spotted as not following [PSR4 standard/convention](https://www.php-fig.org/psr/psr-4/). Namely `ProductQueryFilters` class was in a file named `ProductFiltering.php`. 

This PR renames the file to match the class name - the file path correctly matches the namespace (`Automattic\WooCommerce\Blocks\StoreApi\Utilities`).

This class is used by `ProductCollectionData` from Store API. In particular, `calculate_price_range` and `calculate_attribute_counts` which query the product data by product attribute and price range. These are used to power the _Filter by Attribute_ and _Filter by Price_ blocks (used with _All Products_).

### How to test the changes in this Pull Request:
1. Publish a page with _All Products_, _Filter by Attribute_ and _Filter by Price_ blocks.
2. Set up product data with attributes and pricing so you can test the filters :)
3. Clone this branch. 
4. `composer install` to refresh autoloaded PHP classes.
5. On the front end, test your all products page works correctly, and all filters work as expected.
